### PR TITLE
[EDU-4101] - add whoami command docs

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue: https://aziontech.atlassian.net/browse/EDU-4101

Changes: 

- add whoami command docs

### CODE REVIEW

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_whoami
+menu_namespace: cliMenuAlpha
+permalink: /documentation/devtools/cli/whoami/
+---
+
+## whoami
+
+Shows the email of the currently logged in user on Azion CLI.
+
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx:13]:
```suggestion

Shows the email of the currently logged in user on Azion CLI.

---

```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/whoami/whoami.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_whoami
+menu_namespace: cliMenuAlpha
+permalink: /documentacao/devtools/cli/whoami/
+---
+
+## whoami
+
+Exibe o email do usuário que está logado na Azion CLI.
+
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/whoami/whoami.mdx:13]:
```suggestion
Exibe o email do usuário que está logado na Azion CLI.

---
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Azion CLI whoami
+description: Azion CLI whoami command displays the email of the user currently logged in.
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_whoami
+menu_namespace: cliMenuAlpha
+permalink: /documentation/devtools/cli/whoami/
+---
+
+Shows the email of the currently logged in user on Azion CLI.
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx:10]: Displays the email of the currently logged-in user on Azion CLI.

File: src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+```bash
+$ azion whoami
+```
+
+## Local options
+
+### help
+
+The `-h` or `--help` flag displays more information about the `azion whoami` command. It is an optional flag.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli/commands/whoami/whoami.mdx:28]:
```suggestion
The `-h` or `--help` flag displays more information about the `azion whoami` command. It's an optional flag.
```

File: src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/whoami/whoami.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+Exibe o email do usuário que está logado na Azion CLI.
+
+:::note
+Se um ou mais campos obrigatórios não forem informados através das flags específicas, uma mensagem interativa será exibida, solicitando essas informações.
+:::
+
+---
+## Uso
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/devtools/cli/azion-cli/comandos/whoami/whoami.mdx:17]:
```suggestion

## Uso
```

**Pull request merged**